conf/config.toml

```diff
@@ -68,28 +66,36 @@ region-schedule-limit = 64
 replica-schedule-limit = 64
 merge-schedule-limit = 8
 hot-region-schedule-limit = 4
-#tolerant-size-ratio = 0.0
-#enable-one-way-merge = false
-
-# customized schedulers, the format is as below
-# if empty, it will use balance-leader, balance-region, hot-region as default
+## When the score difference between the leader or Region of the two stores is 
```

Comment:
Do we need to replace "Region" with "region"?

Reply:
In fact, we need to replace all "region" with "Region"